### PR TITLE
[vsphere]: Update README with limitations in Virtual Machine metrics

### PR DESCRIPTION
**Pull request by ishleenk17**
- Bug

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates https://github.com/elastic/sdh-beats/issues/3977

### CODE REVIEW

File: packages/vsphere/docs/README.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ To access the metrices, the url https://host:port(8989)/sdk needs to be passed t
 
  The virtual machine consists of a set of specification and configuration files and is backed by the physical resources of a host. Every virtual machine has virtual devices that provide the same functionality as physical hardware but are more portable, secure and easier to manage.
 
+ Note: vSphere Integration currently supports network names of VM's conecected only to vSS (Virtual Standard Switch) and not vDS (virtual Dustributed Switches).
```
Comment (shmsr) [inline packages/vsphere/docs/README.md:16]:
```suggestion
> Note: vSphere Integration currently supports network names of VMs connected only to vSS (Virtual Standard Switch) and not vDS (virtual Distributed Switches).
```

Notes are usually in blockquote. Also spelling mistakes ^

File: packages/vsphere/_dev/build/docs/README.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ To access the metrices, the url https://host:port(8989)/sdk needs to be passed t
 
  The virtual machine consists of a set of specification and configuration files and is backed by the physical resources of a host. Every virtual machine has virtual devices that provide the same functionality as physical hardware but are more portable, secure and easier to manage.
 
+ Note: vSphere Integration currently supports network names of VM's conecected only to vSS (Virtual Standard Switch) and not vDS (virtual Dustributed Switches).
```
Comment (shmsr) [inline packages/vsphere/_dev/build/docs/README.md:16]:
```suggestion
> Note: vSphere Integration currently supports network names of VMs connected only to vSS (Virtual Standard Switch) and not vDS (virtual Distributed Switches).
```

Notes are usually in blockquote. Also spelling mistakes ^

**Review by shmsr — CHANGES_REQUESTED** [review]

File: packages/vsphere/_dev/build/docs/README.md (hunk 2 above)
Comment (devamanv) [inline packages/vsphere/_dev/build/docs/README.md:16]: Question: Is vSS vSphere Standard Switch or Virtual Standard Switch? Or are they interchangeable? I referred this https://www.vmware.com/in/products/vsphere/distributed-switch.html

Comment (shmsr) [reply packages/vsphere/_dev/build/docs/README.md]: ++

Same Q. I see `v` is `vSphere` (not virtual) in all the places I have checked.

Comment (ishleenk17) [reply packages/vsphere/_dev/build/docs/README.md]: Thanks for pointing this out. My latest change didn't get committed properly from repo. 
Its updated now.

File: packages/vsphere/_dev/build/docs/README.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ To access the metrices, the url https://host:port(8989)/sdk needs to be passed t
 
  The virtual machine consists of a set of specification and configuration files and is backed by the physical resources of a host. Every virtual machine has virtual devices that provide the same functionality as physical hardware but are more portable, secure and easier to manage.
 
+ Note: vSphere Integration currently supports network names of VM's conecected only to vSS (VSphere Standard Switch) and not vDS (VSphere Dustributed Switches).
```
Comment (shmsr) [inline packages/vsphere/_dev/build/docs/README.md:16]:
```suggestion
 Note: vSphere Integration currently supports network names of VMs connected only to vSS (vSphere Standard Switch) and not vDS (vSphere Distributed Switches).
```

File: packages/vsphere/changelog.yml [hunk 4]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 1.9.1
+  changes:
+    - description: Update the README with limitations in Virtual Machine metrics.
+      type: bugfix
```
Comment (devamanv) [inline packages/vsphere/changelog.yml:5]: Would it make sense to treat it an enhancement? Or is it related to a bugfix, that's why we are calling it a bugfix?

**Review by devamanv — APPROVED** [review]
Just one small comment about the update being an enhancement, rest looks good to me.

File: packages/vsphere/changelog.yml (hunk 4 above)
Comment (ishleenk17) [reply packages/vsphere/changelog.yml:5]: Its an addition of a missing documentation, so bugfix should be good

**Review by shmsr — APPROVED** [review]

**Pull request merged**